### ISSUE TITLE
Support Data Sources for Service Quotas

### ISSUE THREAD

**Issue by bflad** [terraform-providers/terraform-provider-aws#9122]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The new Service Quotas service provides lookups for existing service quotas. Let's support those lookups via new data source(s).

For managing service quotas, see #9118 

### New Data Source

* `aws_servicequotas_service`
* `aws_servicequotas_service_quota`

### Potential Terraform Configuration

```hcl
# Relevant API Calls:
#   ListServicesPages
# Additional attributes:
#   service_code # TypeString, Computed
data "aws_servicequotas_service" "example" {
  service_name  = "..." # TypeString, Required
}

# Relevant API Calls:
#   GetServiceQuota
#   ListServiceQuotasPages
# Additional attributes:
#   value # TypeFloat, Computed
data "aws_servicequotas_service_quota" "example" {
  quota_code    = "..." # TypeString, Optional, ConflictsWith quota_name
  quota_name    = "..." # TypeString, Optional, ConflictsWith quota_code
  service_code  = "..." # TypeString, Required
}
```

### References

* https://aws.amazon.com/about-aws/whats-new/2019/06/introducing-service-quotas-view-and-manage-quotas-for-aws-services-from-one-location/
* https://aws.amazon.com/blogs/mt/introducing-service-quotas-view-and-manage-your-quotas-for-aws-services-from-one-central-location/
* `GetServiceQuota`
* `ListServicesPages`
* `ListServiceQuotasPages`

**Comment by bflad** [comment]
Pull request submitted: #9177

**Comment by bflad** [comment]
The following has been merged and will release with version 2.18.0 of the Terraform AWS Provider, likely later this week. 👍 

* **New Data Source:** `aws_servicequotas_service`
* **New Data Source:** `aws_servicequotas_service_quota`

**Comment by bflad** [comment]
This has been released in [version 2.18.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2180-july-05-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

For further feature requests or bug reports with this functionality, please create a [new GitHub issue](https://github.com/terraform-providers/terraform-provider-aws/issues/new/choose) following the template for triage. Thanks!
